Tests/AppTests/Common/Helpers/Controllers/Registration/ApproveHelper.swift

```diff
@@ -0,0 +1,61 @@
+import Vapor
+import Foundation
+@testable import App
+
+//swiftlint:disable superfluous_disable_command
+//swiftlint:disable force_try
+final class ApproveHelper {
+  
+  @discardableResult
+  static func store(places: [Place] = [], cities: [City] = []) throws -> [App.Event]? {
+    
+    var cityId = [Identifier]()
+    var placeId = [Identifier]()
+    var events = [App.Event]()
+    let iterations: (min: Int, max: Int) = (min: 1, max: Int.random(min: 10, max: 20))
```

Comment:
may be instead of `iterations`? ?
`let randomRange = 1...Int.random(min: 10, max: 20)`
in loop: 
`for _ in randomRange {`

Reply:
sure It's more elegantly!